content/en/getting_started/integrations/aws.md

```diff
@@ -0,0 +1,139 @@
+
+## Setup
+
+1. From the AWS tile on the [Integrations Page][7] in your Datadog account. Select the CloudFormation Template option:
+{{< img src="getting_started/integrations/integration-tile-setup.png" alt="Datadog integrations AWS tile">}}
+
+2. This opens the AWS Console and loads the CloudFormation stack. For this demonstration, the API key has been entered directly into the `DdApiKey` field, but you can also store this value in [Secrets Manager][8] and specify the ARN of the secret:
+{{< img src="getting_started/integrations/cloudformation-options-1.png" alt="First part of CloudFormation options for creating Datadog stacks">}}
+
+3. Ensure that the the correct [Datadog site][9] parameter is used for the `DdSite` parameter to match the region of your Datadog account. Check the required boxes from AWS and click `Create stack`:
+{{< img src="getting_started/integrations/cloudformation-options-2.png" alt="Second part of CloudFormation options for creating Datadog stacks">}}
+
+
+This begins the creation process for the Datadog stack along with three nested stacks:
+{{< img src="getting_started/integrations/cloudformation-stacks-complete.png" alt="Datadog stacks completed in CloudFormation">}}
+
+4. From the integration tile on the Datadog side, enter the AWS account ID and AWS Role name. If the role name wasn't changed in the CloudFormation setup page, use `DatadogIntegrationRole`. Datadog automatically queries AWS with the provided values, and a successful query returns the following status:
+{{< img src="getting_started/integrations/integration-tile-status.png" alt="Datadog AWS integration tile with 'Account credentials are valid' message">}}
+
+5. Finally, click `Install integration`. For the purposes of this demonstration, all options have been selected (only `Collect custom metrics` is unticked by default):
+{{< img src="getting_started/integrations/integration-tile-complete.png" alt="Datadog AWS integration tile">}}
+
+It should be noted that there are multiple options within the integration tile to specify the data to be collected. This can be limited based on AWS service, namespace, or tags.
+With [EC2 automuting][10] enabled, Datadog uses host statuses reported by the CloudWatch API to distinguish between expected and unexpected instance terminations, preventing unwanted Monitor notifications.
+
+### Validation
+
+The automatically-provisioned [AWS overview dashboard][11] provides an immediate window into the metrics sent by your AWS services and infrastructure:
+{{< img src="getting_started/integrations/aws-dashboard.png" alt="AWS overview dashboard">}}
+
+
+## Sending Logs
+
+As a result of setting up the Datadog Forwarder function with the CloudFormation template, log collection can be configured from directly within the integration tile in Datadog. Click into the `Collect Logs` tab in the Integration tile and enter the ARN of the Forwarder lambda. With the ARN entered, click the check mark to establish the connection: 
```

Comment:
```suggestion
As a result of setting up the Datadog Forwarder function with the CloudFormation template, log collection can be configured directly in the integration tile in Datadog. Click into the `Collect Logs` tab in the Integration tile and enter the ARN of the Forwarder Lambda. With the ARN entered, click the checkmark to establish the connection: 
```

Reply:
As a result of setting up the Datadog Forwarder function with the CloudFormation template, you can configure log collection directly in the integration tile in Datadog. Click into the `Collect Logs` tab in the Integration tile and enter the ARN of the Forwarder Lambda. With the ARN entered, click the checkmark to establish the connection: